update-readme.R

```diff
@@ -5,7 +5,7 @@ update_readme <- function(png_dir = here::here("man", "figures")){
   pkg_tar_dir <- withr::local_tempdir("mpn-scorecard-")
 
   # Download example package tarball (can change this to MPN snapshot later if desired)
-  pkg_tar <- download.packages("nmrec", destdir = pkg_tar_dir, repos = "https://s3.amazonaws.com/mpn.metworx.dev/releases/nmrec/0.1.0")
+  pkg_tar <- download.packages("nmrec", destdir = pkg_tar_dir, repos = "https://s3.amazonaws.com/mpn.metworx.dev/releases/nmrec/0.2.0")
```

Comment:
No objections to this update, but just to make sure I'm not missing something: there's no real reason to update this to the latest nmrec version, right?  (And general it seems to me best to just leave it be to avoid noise in both this script  and the png diffs.)

Also, since nmrec is now on MPN, I'd say it makes sense to update this to use https://mpn.metworx.com/snapshots/stable/2023-09-19 instead of the dev URL.

Reply:
Yeah I thought about this - I ultimately decided to bump the version because of the R 4.0 bug that was addressed in `nmrec 0.2.0` just as a precaution. Dont think there's any sense in updating this every release. I'll change it to point to MPN, though the readme should stay the same